pandas/tests/frame/methods/test_replace.py

```diff
@@ -890,9 +890,6 @@ def test_replace_input_formats_scalar(self):
             assert return_value is None
         tm.assert_frame_equal(result, expected)
 
-    def test_replace_limit(self):
```

Comment:
probably was a placeholder reminding author to implement tests using limit kwd?  do we have decent coverage?

Reply:
assuming yes, LGTM

Reply:
Good point. There are no tests in this file that test limit. Added `# TODO` here